v3/package.json

```diff
@@ -83,95 +83,97 @@
-    "@typescript-eslint/eslint-plugin": "^5.48.0",
-    "@typescript-eslint/parser": "^5.48.0",
+    "@typescript-eslint/eslint-plugin": "^5.55.0",
+    "@typescript-eslint/parser": "^5.55.0",
     "@welldone-software/why-did-you-render": "^7.0.1",
-    "autoprefixer": "^10.4.13",
-    "caniuse-lite": "^1.0.30001441",
+    "autoprefixer": "^10.4.14",
+    "caniuse-lite": "^1.0.30001467",
     "clean-webpack-plugin": "^4.0.0",
     "css-loader": "^6.7.3",
     "cypress": "^11.2.0",
-    "eslint": "^8.31.0",
+    "eslint": "^8.36.0",
     "eslint-config-react": "^1.1.7",
-    "eslint-import-resolver-typescript": "^3.5.2",
+    "eslint-import-resolver-typescript": "^3.5.3",
     "eslint-plugin-chai-friendly": "^0.7.2",
     "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^27.2.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jest": "^27.2.1",
     "eslint-plugin-json": "^3.1.0",
-    "eslint-plugin-react": "^7.31.11",
+    "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-testing-library": "^5.9.1",
-    "eslint-webpack-plugin": "^3.2.0",
+    "eslint-plugin-testing-library": "^5.10.2",
+    "eslint-webpack-plugin": "^4.0.0",
     "file-loader": "^6.2.0",
+    "fs-extra": "^11.1.1",
     "html-webpack-plugin": "^5.5.0",
     "identity-obj-proxy": "^3.0.0",
     "istanbul-lib-coverage": "^3.2.0",
-    "jest": "^29.3.1",
+    "jest": "^29.5.0",
     "jest-canvas-mock": "^2.4.0",
-    "jest-environment-jsdom": "^29.3.1",
+    "jest-environment-jsdom": "^29.5.0",
     "json5-loader": "^4.0.1",
-    "mini-css-extract-plugin": "^2.7.2",
+    "mini-css-extract-plugin": "^2.7.5",
     "npm-run-all": "^4.1.5",
     "patch-package": "^6.5.1",
-    "postcss-loader": "^7.0.2",
+    "postcss-loader": "^7.1.0",
     "resize-observer-polyfill": "^1.5.1",
-    "sass": "^1.57.1",
+    "sass": "^1.59.3",
     "sass-loader": "^13.2.0",
     "script-loader": "^0.7.2",
-    "style-loader": "^3.3.1",
-    "ts-jest": "^29.0.3",
+    "style-loader": "^3.3.2",
+    "ts-jest": "^29.0.5",
     "ts-loader": "^9.4.2",
     "ts-node": "^10.9.1",
-    "typescript": "^4.9.4",
+    "typescript": "~4.9.5",
     "wait-on": "^7.0.1",
-    "webpack": "^5.75.0",
+    "webpack": "^5.76.2",
     "webpack-cli": "^5.0.1",
-    "webpack-dev-server": "^4.11.1"
+    "webpack-dev-server": "^4.13.0"
   },
   "dependencies": {
-    "@chakra-ui/react": "^2.4.6",
-    "@dnd-kit/core": "^6.0.7",
-    "@emotion/react": "^11.10.5",
-    "@emotion/styled": "^11.10.5",
+    "@chakra-ui/react": "^2.5.2",
+    "@dnd-kit/core": "^6.0.8",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
     "clsx": "^1.2.1",
-    "d3": "^7.8.0",
+    "d3": "^7.8.2",
     "d3-v6-tip": "^1.0.9",
     "escape-string-regexp": "^5.0.0",
     "expr-eval": "^2.0.2",
-    "framer-motion": "^8.1.7",
+    "framer-motion": "^10.6.0",
```

Comment:
I doublechecked because of the jump in major versions- it doesn't look like this is imported anywhere in v3. Can we remove it?

Reply:
Good question! `framer-motion` and `@emotion/*` were added following the [@chakra-ui installation instructions](https://chakra-ui.com/getting-started), but since they're listed as `peerDependencies`, they're added automatically, so there's no reason for us to list them explicitly among our dependencies. I've removed them in the most recent commit.